docs/data/charts/overview/overview.md

```diff
@@ -16,21 +16,21 @@ The `@mui/x-charts` is an MIT library for rendering charts relying on [D3.js](ht
 And, like other MUI X components, charts are production-ready components that integrate smoothly into your app.
 
 With a high level of customization, MUI X Charts provides on three levels of customization layers: **single components** with great defaults, extensive **configuration props**, and **subcomponents** for flexible composition.
-Additionally, you can also use all the MUI System tools, such as the theme override or the `sx` prop.
+Additionally, you can also use all the [MUI System](https://mui.com/system/getting-started/) tools, such as the theme override or the `sx` prop.
 
 ## Using the documentation
 
-The MUI X Charts documentation has a slightly different structure than other MUI components.
+The MUI X Charts documentation has a slightly different structure than other MUI X components.
```

Comment:
Per https://github.com/mui/mui-x/pull/11556#discussion_r1439903948, moving toward never referencing MUI in the docs.